module/cmd/gravity/cmd/genaccounts.go

```diff
@@ -45,7 +45,10 @@ contain valid denominations. Accounts may optionally be supplied with vesting pa
 			serverCtx := server.GetServerContextFromCmd(cmd)
 			config := serverCtx.Config
 
-			homeFlag, _ := cmd.Flags().GetString(flags.FlagHome)
+			homeFlag, errHomeFlag := cmd.Flags().GetString(flags.FlagHome)
+			if errHomeFlag != nil {
+				fmt.Printf("Homeflag has an error %v", errHomeFlag)
```

Comment:
I think the home directory not being parsed correctly is a reason to panic, thoughts @jkilpatr ?

This would probably help prevent needless questions in the discord if the server doesn't even start with the helpful message.